modules/nw-multus-host-device-object.adoc

```diff
@@ -39,14 +56,31 @@ parameters: `device`, `hwaddr`, `kernelpath`, or `pciBusID`.
   "pciBusID": "<pciBusID>" <5>
 }
 ----
-<1> `name`: Specify the name of the `NetworkAttachmentDefinition`.
+<1> Specify the value for the `name` parameter you provided previously for
+the CNO configuration.
 
-<2> `device`: Specify the name of the device, such as `eth0`.
+<2> Specify the name of the device, such as `eth0`.
 
-<3> `hwaddr`: Specify the device hardware MAC address.
+<3> Specify the device hardware MAC address.
 
-<4> `kernelpath`: Specify the Linux kernel device path, such as
+<4> Specify the Linux kernel device path, such as
 `/sys/devices/pci0000:00/0000:00:1f.6`.
 
-<5> `pciBusID`: Specify the PCI address of the network device, such as
-`0000:00:1f.6`.
+<5> Specify the PCI address of the network device, such as `0000:00:1f.6`.
+
+[id="nw-multus-hostdev-config-example_{context}"]
+== host-device configuration example
+
+The following example configures an additional network named `hostdev-net`:
+
+[source,yaml]
+----
+name: hostdev-net
+type: Raw
+rawCNIConfig: '{ <1>
+  "cniVersion": "0.3.1",
+  "type": "host-device",
+  "device": "eth1"
+}'
+----
+<1> The CNI configuration object is specified as a YAML string.
```

Comment:
Same here: You said above the CNI plug-in configuration is JSON format??

Reply:
@mburke5678, yeah, the plug-in configuration is not very intuitive. The configuration is a JSON object, but it's provided as a _string_ value for a YAML parameter, and then the CNO injects it to the plug-in during initialization.

That's where the single quotes at the beginning and end of the multi-line value come from. I wonder if providing this as a YAML multi-line string might make it more obvious?

Also: https://yaml-multiline.info